exercises/linked-list/dune-project

```diff
@@ -0,0 +1,2 @@
+(lang dune 1.1)
+(version 1.3.0)
```

Comment:
Since this exercise [does not have a canonical version](https://github.com/exercism/problem-specifications/tree/master/exercises/linked-list), we should remove this line.
```suggestion
```